cmd/kube-rbac-proxy/app/options/proxyoptions.go

```diff
@@ -17,10 +17,16 @@ limitations under the License.
 package options
```

Comment:
Nit: use capital letters at the beginning of a sentence, even in commit messages.

Reply:
As it is a nit and it is not targeting anything, I see no CTA.